app/settings.py

```diff
@@ -31,6 +31,12 @@
 SERVICE_REQUEST_TOTAL_RETRIES = 5
 SERVICE_REQUEST_BACKOFF_FACTOR = 0.1
 
+ANTI_VIRUS_ENABLED = bool(os.getenv("ANTI_VIRUS_ENABLED", "True"))
+ANTI_VIRUS_BASE_URL = os.getenv("ANTI_VIRUS_BASE_URL", "https://scan.metadefender.com/v2/")
+ANTI_VIRUS_API_KEY = os.getenv("OPSWAT_API_KEY")
```

Comment:
Using OPSWAT is inconsistent with the other variables, should this be `ANTI_VIRUS_API_KEY`?